prose/fields.py

```diff
@@ -50,10 +50,11 @@ def formfield(self, **kwargs):
 
     def pre_save(self, model_instance, add):
         raw_html = getattr(model_instance, self.attname)
-        sanitized_html = bleach.clean(
-            raw_html, tags=ALLOWED_TAGS, attributes=ALLOWED_ATTRIBUTES
-        )
-        return sanitized_html
+        if raw_html:
+            sanitized_html = bleach.clean(
+                raw_html, tags=ALLOWED_TAGS, attributes=ALLOWED_ATTRIBUTES
+            )
+            return sanitized_html
```

Comment:
```suggestion
        if not raw_html:
            return raw_html

        sanitized_html = bleach.clean(
            raw_html, tags=ALLOWED_TAGS, attributes=ALLOWED_ATTRIBUTES
        )
        return sanitized_html
```

Minor change to utilize early returns and handle both `""` and `None` - previously it would return `None` in both cases.